### PR TITLE
fix sass division warning

### PR DESCRIPTION
**Pull request by rav3**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@govtechsg/sgds",
+  "name": "sgds-govtech",
   "version": "1.3.23",
   "description": "Design System unites Singapore Government digital services around a common visual language and user experience",
```
Comment (clukhei) [inline package.json:4]: verison 1.3.23 already exists on npm sgds-govtech. Should bumped up to 1.3.24 to publish

File: package.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@govtechsg/sgds",
+  "name": "sgds-govtech",
```
Comment (clukhei) [inline package.json:2]: confirm is sgds-govtech and not @govtechsg/sgds-govtech ah ?

Comment (rav3) [reply package.json:2]: yep confirm for now

File: package.json (hunk 1 above)
Comment (rav3) [reply package.json:4]: bumped to 1.3.24

**Pull request merged**